spec.html

```diff
@@ -23791,7 +23791,8 @@ <h1>Runtime Semantics: Evaluation</h1>
             1. Assert: _received_.[[Type]] is ~return~.
             1. Let _return_ be ? GetMethod(_iterator_, *"return"*).
             1. If _return_ is *undefined*, then
-              1. If _generatorKind_ is ~async~, set _received_.[[Value]] to ? Await(_received_.[[Value]]).
+              1. If _generatorKind_ is ~async~, then
+                1. Return Completion Record { [[Type]]: _received_.[[Type]], [[Value]]: ? Await(_received_.[[Value]]), [[Target]]: _received_.[[Target]] }.
```

Comment:
```suggestion
                1. Return Completion Record { [[Type]]: ~return~, [[Value]]: ? Await(_received_.[[Value]]), [[Target]]: ~empty~ }.
```
since we know the values of those fields.

Reply:
I chose the other suggestion.